.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+
+# Summary of Changes
+
+Describe the changes or improvements made in this pull request.
+
+## Related Issue
+
+Closes #<issue-number>
+
+## Checklist
+
+- [ ] I have read and followed the [project's contribution guidelines](https://github.com/secure-systems-lab/lab-guidelines/blob/master/dev-workflow.md), including code style and commit message conventions.
```

Comment:
I think these our not our contributions guidelines

Reply:
The secure-systems-lab is out of the NYU team that leads Uptane. It may be more advantageous to copy the guidelines to our own site and link directly.